### DISCUSSION TITLE
Higher PET and Trmt arount the trees

### DISCUSSION THREAD

**Discussion by VladimirSobral** [UMEP-dev/UMEP#579] [Q&A]
Hi,

I used the SpatialTC to generate a PET map of my neighbourhood. However, I noticed that the highest PET values are around the trees (as you can see in the figure attached). For me, these higher values ​​should be close to buildings, especially on facades directly facing the sun. Another point is that the Trmt map shows similar information (higher Trmt values ​​around trees).

Am I using some wrong configuration in the trees?
Can anyone help me understand this?

Thanks,

![Test_PET](https://github.com/UMEP-dev/UMEP/assets/124792761/d818321a-cb55-4612-b6f2-4b80543bf3df)

**Comment by nilswallenberg** [comment]
Hi,

Can you please share your data (Tmrt raster, wind raster and met file).

Cheers,
Nils

**Reply by VladimirSobral** [reply]
Sure,

Please let me know if you need more information.

Thanks,
[JardimOceania_metfile.zip](https://github.com/UMEP-dev/UMEP/files/13685262/JardimOceania_metfile.zip)

**Comment by nilswallenberg** [comment]
Maybe if possible attach the entire output from SOLWEIG

**Reply by VladimirSobral** [reply]
Yes, all the files are here.

Thanks,
[QGIS.zip](https://github.com/UMEP-dev/UMEP/files/13685840/QGIS.zip)

**Reply by nilswallenberg** [reply]
You ran SOLWEIG with UTC 0. Try to run it with UTC-3 (which I think is the correct UTC for your location). Hopefully it will work. :)

**Reply by VladimirSobral** [reply]
Hi,

Thank you so much for your reply.
I ran SOLWEIG as you suggested, and the result seems to be more appropriate for the region (figure 1).
Please let me ask two questions:
1. Regarding the PET map, I still found a few higher PET values ​​located under trees in areas facing the sun (WEST). Please see figure 2. I am considering the height of the tree trunk as 2 meters, which implies a canopy relatively close to a person underneath it. Could this imply an increase in PET?
2. I noticed that the WALL_HEIGHT raster had values ​​from 0 to more than 10000.00 (figure 3). However, the tallest building in my region reaches approximately 100m. What could be wrong with the input data?

Thanks,
Vlad

![1](https://github.com/UMEP-dev/UMEP/assets/124792761/b5c67745-806f-48b7-8213-26a23da9fef5)
![2](https://github.com/UMEP-dev/UMEP/assets/124792761/224dd2c4-d9ef-4a90-849f-1ee5f06669f7)
![3](https://github.com/UMEP-dev/UMEP/assets/124792761/46841ccf-f8fa-469b-b4a1-e8f01e91f3bd)

**Reply by biglimp** [reply]
1. Yes, this could actualy be the case since you still have solar radiation under the tree but at the same time, the tree canopy is blocking the "cool" sky from incoming longwave radiation¨
2. This is probably because you have no data values outside of your area (-9999) which will be a gigantic wall. It will not affect your result within your area but you can replace these nodata value with e.g. 0 to get nicer results.

**Reply by VladimirSobral** [reply]
perfect! thanks @biglimp
